src/containers/Home/Postlist/index.js

```diff
@@ -7,10 +7,15 @@ import './style.scss';
 
 export class Postlist extends React.Component {
   static propTypes = {
-    post: PropTypes.object,
+    post: PropTypes.shape,
     vote: PropTypes.func,
```

Comment:
`vote` is a dispatcher given by `mapDispatchToProps`, which means it would must be there.
so you can simply add `.isRequired` to avoid warning.